_posts/2022-02-23-paying-contributors-sponsoring-projects.md

```diff
@@ -0,0 +1,102 @@
+* [Sindre Sorhus](https://sindresorhus.com/) is a prolific open source developer, and ESLint uses several of his modules, including [`chalk`](https://npmjs.com/package/chalk), [`globals`](https://npmjs.com/package/globals), [`import-fresh`](https://npmjs.com/package/import-fresh), [`strip-ansi`](https://npmjs.com/package/strip-ansi), and [`strip-json-comments`](https://npmjs.com/package/strip-json-comments).
+* [debug](https://npmjs.com/package/debug) is a small tool ESLint uses to output debugging messages when you use the `--debug` flag. 
+* [lint-staged](https://npmjs.com/package/lint-staged) is a simple precommit hook manager that makes it easy to setup linting in your development process.
+
+We think it's important for open source projects that receive a lot of donations to take care of their dependencies. After all, if your project couldn't exist without those dependencies, then they deserve your support.
+
+You can always check to see which projects ESLint is supporting on our [Open Collective page](https://opencollective.com/eslint#category-CONTRIBUTIONS).
+
+### Supporting the community
+
+Once we started supporting our upstream dependencies, we though, what about the downstream dependencies? There are a lot of projects related to ESLint that make the ESLint ecosystem better. Whether that be a different set of rules, a custom parser, or anything that makes ESLint better to use. We are currently donating $150 per month to these projects:
+
+* [typescript-eslint](https://github.com/typescript-eslint) is the plugin that makes ESLint work with TypeScript.
+* [eslint-plugin-import](https://npmjs.com/package/eslint-plugin-import) is a plugin to help you manage your module imports.
+* [eslint-plugin-jsx-a11y](https://npmjs.com/package/eslint-plugin-jsx-a11y) is a plugin to help you ensure your JSX code is accessible.
+
+Going forward, we are looking to continue supporting community projects like these. The strength of ESLint is in the ecosystem built by you, and we want to support as much of that ecosystem as possible.
+
+### Big projects
+
+The last way we decided to spend money is on what we playfully called "big projects." These are projects that take a lot of time and effort, and that the team doesn't have the time, interest, or expertise to accomplish on our own. Every project has a backlog of tasks that seem too big to ever accomplish, and we realized that we could start making our way through that list by hiring professionals to do the work.
+
+The first of our big projects is an ambitious project to redesign the ESLint website. We had wanted to redesign our website for several years, and tried at least three separate times to do it in the spirit of open source, which volunteers and designing by consensus. Each of those times the efforts failed, and so we decided it was time to hire some folks to help us out.
```

Comment:
```suggestion
The first of our big projects is an ambitious project to redesign the ESLint website. We had wanted to redesign our website for several years, and tried at least three separate times to do it in the spirit of open source, which involves volunteers and designing by consensus. Each of those times the efforts failed, and so we decided it was time to hire some folks to help us out.
```

Reply:
Looks like I can’t merge this on the GitHub so I’ll have to wait until I get to the computer next.

Reply:
Maybe it is not needed--I hadn't seen Milos' earlier suggestion to change this to `with volunteers and designing by consensus`, so with that merged, this alternative isn't really necessary I think.

Reply:
Ah good call. 👍